tools/jit/templates/register_aten_ops.cpp

```diff
@@ -24,6 +24,15 @@
 
 // ${generated_comment}
 
+// NOTE: This file is generated in a sharded fashion to speed up
```

Comment:
The convention in the codebase is to give a note a name, like

```
// Note [Sharded file]
// This file is blah blah blah
```

And then refer to the note using `See Note [Sharded file]`. This allows devs to easily grep for all sites of a note reference.

Reply:
will do